Documentation/ceph-object-bucket-claim.md

```diff
@@ -105,13 +105,14 @@ provisioner: rook-ceph.ceph.rook.io/bucket [2]
 parameters: [3]
   objectStoreName: my-store
   objectStoreNamespace: rook-ceph
-  region: us-west-1
-  bucketName: ceph-bucket [4]
-reclaimPolicy: Delete [5]
+  region: us-west-1 [4]
+  bucketName: ceph-bucket [5]
+reclaimPolicy: Delete [6]
 ```
 1. `label`(optional) here associates this `StorageClass` to a specific provisioner.
 1. `provisioner` responsible for handling `OBCs` referencing this `StorageClass`.
 1. **all** `parameter` required.
+1. `region` needs to create required zone/zonegroup in RGW before defining it here. Please refer [ceph documentation](https://docs.ceph.com/en/latest/radosgw/multisite/#multi-site) for more details.
```

Comment:
I'm really glad that we are providing better user documentation here. I am still struggling to understand how this is intended to be used, and I think users will likely be much more confused than I.

A good description should give a quick overview of what region is and explain how to fill in the field in relation to other Rook resources? How is "region" related to zone/zonegroup? The statement "needs to create required zone/zonegroup in RGW before defining it here" doesn't use Rook terminology for what needs to be created and so might be confusing for users. Also, what if users aren't using CephObjectZone or CephObjectZoneGroup?

Based on what I "think" the usage pattern intended to be, I would suggest something like below, but I may be misunderstanding some details.
```suggestion
1. `region` defines the region in which the bucket should be created. For Rook, this is CephObjectZoneGroup (if applicable) of the related CephObjectStore; or the name of the CephObjectStore if no CephObjectZoneGroup is defined. Please refer to [Ceph's documentation](https://docs.ceph.com/en/latest/radosgw/multisite/#multi-site) for more details.
```